src/main/kotlin/racingcar/domain/car/Position.kt

```diff
@@ -0,0 +1,14 @@
+package racingcar.domain.car
+
+class Position(val value: Int) {
+
+    constructor() : this(0)
```

Comment:
What about extracting this into a static field?

```suggestion
    constructor() : this(START_POSITION)
```

Reply:
That's a good idea👍🏻
I thought `0` also gives a clear meaning and doesn’t need to be made into a constant, but your recommendation is much more readable and clear!